recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend

```diff
@@ -10,11 +10,6 @@ SRC_URI_append = " \
     file://defconfig \
 "
 
-SRC_URI_append_rcar = " \
```

Comment:
Is dropping these patches somehow M3N specific?

Reply:
No, there should be another patch for dropping this.